contributingGuides/CONTRIBUTING.md

```diff
@@ -15,6 +15,9 @@ You can create as many accounts as needed in order to test your changes directly
 ##### Generating Multiple Test Accounts
 You can generate multiple test accounts by using a `+` postfix, for example if your email is test@test.com, you can create multiple New Expensify accounts connected to the same email address by using test+123@test.com, test+456@test.com, etc.
 
+##### High Traffic Accounts
+All internal engineers, contributors, and C+ members are **required** to test with a "high traffic" account against the staging or production web servers. Ask in `#expensify-open-source` if someone can turn your account into a High Traffic account. These accounts more closely mirror the accounts used in production by real people. Internal team members can follow [this Stack Overflow](https://stackoverflow.com/c/expensify/questions/14504) to upgrade an account.
```

Comment:
Same thing I mentioned above for consistency wrt [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) vs `#expensify-open-source`.